### PR TITLE
[Podspec] Set Swift Version to 4.0.0

### PR DESCRIPTION
**Pull request by Ben-G**
## Changes in this pull request

Pin Swift version in podspec.

### CODE REVIEW

File: ReactiveLists.podspec [hunk 1]
```diff
@@ -15,5 +15,6 @@ Pod::Spec.new do |s|
   s.ios.deployment_target = '9.0'
   s.requires_arc = true
   
+  s.swift_version = '4.0'
   s.dependency 'Dwifft', '~> 0.7.0'
-end
+end
```
Comment (jessesquires) [inline ReactiveLists.podspec]: newline plz 😬

**Comment by jessesquires** [comment]
also, travis will complain about this

we should add CocoaPods to Gemfile

File: ReactiveLists.podspec (hunk 1 above)
Comment (Ben-G) [reply ReactiveLists.podspec]: Haha... sorry

**Comment by Ben-G** [comment]
@jessesquires Feedback addressed!

**Pull request merged**